spec/classes/redis_spec.rb

```diff
@@ -926,7 +926,8 @@
 
           it { is_expected.to contain_file(config_file_orig).with(
```

Comment:
I think it's cleaner do use `it do ... end` (multi line) or `it { is_expected.to contain_file(config_file_orig).with_content(/^slaveof _VALUE_/) }`